test/test.js

```diff
@@ -1906,6 +1906,44 @@ describe('Page', function() {
       const screenshot = await elementHandle.screenshot();
       expect(screenshot).toBeGolden('screenshot-element-padding-border.png');
     });
+    it('should capture full element when larger than viewport', async({page, server}) => {
+      // compare with .to-screenshot size
+      await page.setViewport({width: 500, height: 500});
+
+      await page.setContent(`
+        something above
+        <style>div.spacer {
+          border: 2px solid red;
+          background: red;
+          height: 600px;
+          width: 52px;
+        }
+        div.to-screenshot {
+          border: 2px solid blue;
+          background: rgba(0, 0, 0, 0.5);
+          width: 600px;
+          height: 200.5px;
+          margin-left: 50px;
+          transform: scaleY(1.2);
+        }
```

Comment:
This will show scrollbars in headful mode. To hide them, add:
```css
::-webkit-scrollbar {
  display: none;
}
```

Reply:
Fixed